<launch>
  <node pkg="hrpsys_gazebo_atlas" type="triangle_screenpoint.py" name="triangle_screenpoint"
        output="screen"
        clear_params="true">
  </node>
</launch>
